lib/apiservers/engine/backends/convert/stats.go

```diff
@@ -75,6 +81,8 @@ func NewContainerStats(config *ContainerStatsConfig) *ContainerStats {
 		curDockerStat: &types.StatsJSON{},
 		totalVCHMhz:   uint64(config.VchMhz),
 		dblVCHMhz:     uint64(config.VchMhz * 2),
```

Comment:
It is not related to vCPU count.  I have a comment as to why this is the case when the variable is used:  https://github.com/cgtexmex/vic/blob/stats-4422/lib/apiservers/engine/backends/convert/stats.go#L402